editor.planx.uk/src/pages/FlowEditor/index.tsx

```diff
@@ -25,29 +27,42 @@ interface Operation {
   data: Array<OT.Op>;
 }
 
-export const LastEdited = () => {
-  const [flowId, flow] = useStore((state) => [state.id, state.flow]);
+const formatLastEditDate = (date: string): string => {
```

Comment:
I wondered about using `formatDistanceToNow()` for the initial version (as it's what's used in the last published text).

The reason I decided against it was that it gets stale - it's correct each time it's rendered, but if there's no operations for a while and time passes, it's incorrect.

We could certainly update them by calling `refetch()` on a timeout (30 seconds?). I think also just re-rendering here without a refetch would achieve the same - I'd guess the Apollo cache may make both of these equal but would need to try it.

I'm not sure what the right solution / pattern is here - maybe a good question for @ianjon3s!

Not a blocking issue, but it is one we'll need to address sooner or later I think.

Reply:
Rough implementation of this here - https://github.com/theopensystemslab/planx-new/pull/3059

Likely an issue which can be resolved by UI alone 👍